arm-resources/managedapplications/2016-09-01-preview/swagger/managedapplications.json

```diff
@@ -225,7 +225,6 @@
           {
             "name": "parameters",
             "in": "body",
-            "required": true,
```

Comment:
The properties in the body should also be all optional.
Imagine someone trying to patch an `Appliance`, with this API, that person has to provide `properties` and `kind`, which shouldn't be the case.
Many RPs find adding a `BlahPatchable` class helpful.